apps/io-services-cms-webapp/openapi.yaml

```diff
@@ -132,12 +132,6 @@ paths:
       summary: Retrieve all services
       description: Retrieve all service topics
       operationId: getServiceTopics
-      parameters:
-        - $ref: '#/components/parameters/UserEmail'
-        - $ref: '#/components/parameters/UserGroups'
-        - $ref: '#/components/parameters/UserID'
-        - $ref: '#/components/parameters/SubscriptionID'
-        - $ref: '#/components/parameters/XForwardedFor'
```

Comment:
chore: from the _Services CMS App_ point of view, by removing _XForwardedFor_ (i.e.: avoiding to explicitly setting _XForwardedFor_ header) we "lost" the origin caller IP....Azure will automatically add to this header only the last caller IP (i.e. the _Backoffice App_ IP)

Reply:
added in 07afcb591d3b9e39a6ec23067d6ae333b1c05be7